src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+* {
+  margin: 0;
+}
```

Comment:
Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document